app/registries/foreman/access_permissions.rb

```diff
@@ -9,6 +9,10 @@
       :notification_recipients => [:index, :update, :destroy, :update_group_as_read, :destroy_group ] }, :public => true
     map.permission :api_status, { :"api/v2/home" => [:status]}, :public => true
     map.permission :about_index, { :about => [:index] }, :public => true
+    map.permission :view_table_preferences, { 'api/v2/table_preferences': [:index, :show] }, public: true
```

Comment:
Do we need so many permissions for such a simple resource? :(

Reply:
I'd also be pro refactoring `editing_self?` if possible instead of adding a new permission, users should be able to edit their preferences only and if e.g. admin needs to edit preferences of other users, they can impersonate their account